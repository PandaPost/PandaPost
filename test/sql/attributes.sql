\set ECHO none

\i test/pgxntool/setup.sql

SELECT plan(
  1 -- T
);

-- ediff1d
SELECT is(
    str("T"(eval('[[1,2],[3,4]]')))
    , '[[1 3]
 [2 4]]'
    , 'T([[1,2],[3,4]])'
);


\i test/pgxntool/finish.sql


-- vi: expandtab sw=2 ts=2
