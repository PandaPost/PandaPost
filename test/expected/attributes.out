\set ECHO none
1..1
ok 1 - T([[1,2],[3,4]])
# TRANSACTION INTENTIONALLY LEFT OPEN!
